sdk/identity/identity/src/credentials/usernamePasswordCredentialOptions.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { InteractiveCredentialOptions } from "./interactiveCredential";
+
+/**
+ * Defines options for the {@link UsernamePasswordCredential} class.
+ */
+export interface UsernamePasswordCredentialOptions extends InteractiveCredentialOptions {}
```

Comment:
`InteractiveCredentialOptions` has `authenticationRecord` and `disableAutomaticAuthentiation` which don't apply to the `UsernamePasswordCredential`. I think you could just derive from `PersistentCredentialOptions`

Reply:
Looks like this still derives from `InteractiveCredentialOptions`